src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx

```diff
@@ -604,7 +551,7 @@ const AddReportDrawerContent: React.FC = () => {
                 alt="status value"
```

Comment:
Alt means what this section means, so `status value` might not be suitable here.

Reply:
I changed the alt to "description".